### ISSUE TITLE
specify separate number of threads for meta and compilation phases

### ISSUE THREAD

**Issue by wkotlarski** [FlexibleSUSY/FlexibleSUSY#129]
Would it be possible to have it? Also, we could check if the requested number of cores for the meta phase is `<= $MaxLicenseProcesses`.

**Comment by wkotlarski** [comment]
This would be useful because modern computers typically have much more cores than mathematica licences.

**Comment by pathron** [comment]
Yes thats the normal situation for me.  when it hangs because of the license problem its quite annoying and I would really like to be able to use as many cores as i wish for code compilation without worrying about mathematica license number.
